src/containers/Accounts/AccountAssetTab/AccountAssetTab.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { ChangeEvent } from 'react';
+import { useTranslation } from 'react-i18next';
+import { connect } from 'react-redux';
+import { useHistory, useParams } from 'react-router';
+import { AccountTokenTable } from '../AccountTokenTable';
+import { AccountNFTTable } from '../AccountNFTTable/AccountNFTTable';
+
+// TODO: Add state types or convert to react query
+interface Props {
+  account: any;
```

Comment:
Is there a better type we can use than `any`?

Reply:
That would be good.  The account is so complex I considered out of scope for the current work.  Would like to do a refactor where we at the very least use the type definitions from xrpl.js since we dont need much of the other code for interacting with stuff.

Reply:
I believe there's an outstanding task for us to separate out the typing so that we can reuse it in multiple projects